linux_os/guide/services/mail/postfix_client/postfix_network_listening_disabled/rule.yml

```diff
@@ -42,3 +42,5 @@ ocil: |-
     Run the following command to ensure postfix accepts mail messages from only the local system:
     <pre>$ grep inet_interfaces /etc/postfix/main.cf</pre>
     If properly configured, the output should show only <tt>{{{ xccdf_value("var_postfix_inet_interfaces") }}}</tt>.
+
+platform: postfix
```

Comment:
@marcusburghardt You can remove the platform from the rule.
It will inherit it from the Group it is in.

Reply:
@yuumasato , this rule is in the group `postfix_client` while the modified group is `postfix_harden_os`, in the same level. I can't include the platform parameter in the group `postfix_client`, without change the directories structure, because other rules there actually are not dependent of postfix package, differently of `postfix_harden_os` group, where all rules depend of postfix package.

Reply:
@marcusburghardt It seems to me that all rules in `postfix_client` group only make sense when postfix is installed.
I think `postfix_client_configure_mail_alias` is the least obvious, but the rule and remediation mention `postconf` and `newaliases`, which are `postfix` binaries.

Reply:
Discussed offline with @marcusburghardt , and Rule `postfix_client_configure_mail_alias` is more related to root mail alias then `postfix` itself.